webpack/ForemanRhCloudFills.js

```diff
@@ -33,10 +33,11 @@ const fills = [
   {
     slot: 'host-details-page-tabs',
     name: 'CVEs',
```

Comment:
If this is the tab name in hash, then this should be also updated to reflect the actual tab title ("Vulnerabilities").